extensions/git/src/repository.ts

```diff
@@ -973,6 +973,21 @@ export class Repository implements Disposable {
 		const onDidChangeCountBadge = filterEvent(workspace.onDidChangeConfiguration, e => e.affectsConfiguration('git.countBadge', root));
 		onDidChangeCountBadge(this.setCountBadge, this, this.disposables);
 		this.setCountBadge();
+
+		this.disposables.push(window.onDidEndTerminalShellExecution(e => {
+			const [executable, subcommand, ..._args] = e.execution.commandLine.value.split(' ');
```

Comment:
Need a little bit more careful parsing.

Reply:
Any idea where I should put this or is it fine in `Repository.ctor`?